qiskit/quantum_info/states/random.py

```diff
@@ -31,7 +31,7 @@ def random_statevector(
 ) -> Statevector:
     """Generator a random Statevector.
 
-    The statevector is sampled from the uniform (Haar) measure.
+    The statevector is sampled from the uniform distribution.
```

Comment:
I would also prefer to keep the `Haar` keyword, as that's a common description for this state. How about
```suggestion
    The statevector is sampled from the uniform distribution, also known as Haar-random state.
```

Reply:
Having thought about it more last night, I think it's actually quite important that we include "Haar" - there's no interval over which to be "uniform" without defining some sort of invariant volume. We do that via the volume of the space of unitary matrices that we get from the Haar measure on them, applied to some fixed normalised state.

Reply:
Done.